pkg/docker/config/config.go

```diff
@@ -719,3 +751,20 @@ func normalizeRegistry(registry string) string {
 	}
 	return normalized
 }
+
+// isNamespaced returns true if the provided registry contains one or more
+// subpaths.
+func isNamespaced(registry string) bool {
```

Comment:
The input is not a registry. (Similarly the `registryIsNamespaced` variable name doesn’t make sense.)

Reply:
Function got removed after the next push and refactored into `validateKey`